Infrastructure/Factories/QualityAssuredSpreadsheetTuitionPartnerFactory.cs

```diff
@@ -3,85 +3,89 @@
 using Application.Factories;
 using Domain;
 using Domain.Constants;
+using Domain.Enums;
```

Comment:
The `Domain.Enums` namespace unused. Can we remove this?

Reply:
Agreed, change made.